test/test_turbulence_closures.jl

```diff
@@ -0,0 +1,144 @@
+function test_closure_instantiation(; closures=(
```

Comment:
I feel a more modular/useful test would be to test whether a single closure instantiates properly. E.g.
```julia
@test test_closure_instantiation(:ConstantSmagorinsky, Float64)
```
and so on. So if one closure fails to instantiate but the others pass, then it's easy to isolate exactly which one failed.

Reply:
Ok, I'll make that change.